modules/replicator/src/testFixtures/java/org/apache/ignite/internal/raft/client/AbstractTopologyAwareGroupServiceTest.java

```diff
@@ -261,26 +259,24 @@ private IgniteBiTuple<TopologyAwareRaftGroupService, TopologyAwareRaftGroupServi
 
         afterClusterInit(leader.name());
 
+        raftClients.add(raftClient);
```

Comment:
I would move this to the `org.apache.ignite.internal.raft.client.AbstractTopologyAwareGroupServiceTest#startCluster`

because in the test `org.apache.ignite.internal.raft.client.AbstractTopologyAwareGroupServiceTest#testOneNodeReplicationGroup` we do not add created client to `raftClients`, hence we do not shutdown it

Reply:
It was just a mistack. We can't move the cluster start before startup because the number of nodes is defined by a test.